pandas/tests/frame/test_convert_to.py

```diff
@@ -328,3 +328,26 @@ def test_to_dict_index_dtypes(self, into, expected):
         result = DataFrame.from_dict(result, orient='index')[cols]
         expected = DataFrame.from_dict(expected, orient='index')[cols]
         tm.assert_frame_equal(result, expected)
+
+    def test_to_records_with_multiindex(self):
```

Comment:
I guess I could remove this test `test_to_records_with_multiindex` since `test_to_records_index_name` is already there on line 130 and deals with multi index?